SPECS/enchant/enchant.spec

```
@@ -1,41 +1,41 @@
+Summary:        An Enchanting Spell Checking Library
+Name:           enchant
+Version:        1.6.0
+Release:        28%{?dist}
+License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Summary: An Enchanting Spell Checking Library
-Name: enchant
-Version: 1.6.0
-Release: 27%{?dist}
-License: LGPLv2+
-Source: http://www.abisource.com/downloads/enchant/%{version}/enchant-%{version}.tar.gz
-URL: http://www.abisource.com/
+URL:            https://www.abisource.com/
+Source:         ftp://ftp.netbsd.org/pub/pkgsrc/distfiles/enchant-1.6.0.tar.gz
```

Please make similar updates wherever possible in the moved specs:

```suggestion
Source:         ftp://ftp.netbsd.org/pub/pkgsrc/distfiles/%{name}-%{version}.tar.gz
```